### ISSUE TITLE
Referencing packages in project

### ISSUE THREAD

**Issue by aaroneden** [aws/aws-toolkit-jetbrains#1910]
**Mac OS Catalina 10.15.5**
- JetBrains product: PyCharm Community 2020.1
- AWS Toolkit version: 1.16-201
- SAM CLI version: 0.53.0
- JVM/Python version: 3.7.3

**Question**
How do I go about ensuring my local packages get pulled into the SAM build when I do local debugging of a Lambda?

I have some supporting modules in a modules folder:
<img width="1499" alt="Screen Shot 2020-07-13 at 11 48 44 AM" src="https://user-images.githubusercontent.com/21063939/87348974-79adce80-c50a-11ea-9504-798286fa4b54.png">

I would like for them to get copied in the build process.  I've tried:

1) Adding a modules hack to the YAML, but that doesn't get them:

`  Modules:
    Type: AWS::Serverless::Function
    Properties:
      Runtime: python3.7
      CodeUri: /modules
      Handler: modules.lambda_handler

  ModulesLayer:
    Type: AWS::Serverless::LayerVersion
    Properties:
      LayerName: ModulesLayer
      Description: Dependencies for the blank sample app.
      ContentUri: /.aws-sam/build/modules
      CompatibleRuntimes:
        - python3.7
      RetentionPolicy: Delete
    DependsOn:
      - Modules`

2) Adding a "Before Launch" step into the Run/Debug Configurations to copy my modules into the .aws-sam build folder, but this command is executed and then SAM empties the build folder (including the files I just copied in).

Any suggestions to get me unstuck would be much appreciated.

**Comment by abrooksv** [comment]
SAM build will use what is in the requirements.txt to build the Lambda payload.

Have you tried putting a relative path in it?

**Comment by aaroneden** [comment]
> SAM build will use what is in the requirements.txt to build the Lambda payload.
> 
> Have you tried putting a relative path in it?

Good idea @abrooksv  -- I cannot put a relative path, but I tried adding  "modules==1.0" to the requirements.txt and it looks like the build created a modules-1.0.0.dist-info folder with METADATA,RECORD,tol_level.txt and WHEEL in it.  It looks like it tried to download the package from PyPy or something perhaps?

**Comment by abrooksv** [comment]
Try `../modules`? I'm not very familiar with the Python ecosystem.

**Comment by aaroneden** [comment]
> Try `../modules`? I'm not very familiar with the Python ecosystem.

Thanks for trying to help :)  I tried ../modules, /../modules, /modules with similar results.

**Comment by abrooksv** [comment]
Are you always wanting to make it a layer, or was that a workaround attempt?

**Comment by aaroneden** [comment]
It was a workaround attempt, just trying to keep from repeating source code files across projects :)
